realm/realm-library/src/main/cpp/io_realm_internal_OsRealmConfig.cpp

```diff
@@ -327,4 +327,13 @@ JNIEXPORT void JNICALL Java_io_realm_internal_OsRealmConfig_nativeSetSyncConfigS
     CATCH_STD()
 }
 
+JNIEXPORT void JNICALL Java_io_realm_internal_OsRealmConfig_nativeSetForceSyncHistory(JNIEnv*, jclass,
+                                                                                      jlong native_ptr)
+{
+    TR_ENTER_PTR(native_ptr);
+
+    auto& config = *reinterpret_cast<Realm::Config*>(native_ptr);
+    config.force_sync_history = true;
```

Comment:
```
        // FIXME: Realm Java manages sync at the Java level, so it needs to create Realms using the sync history
        //        format.
        bool force_sync_history = false;
```

I thought this should be removed? It was added because of we were handling sync in java at the beginning.